modules/rollback-central-forced.adoc

```diff
@@ -15,7 +15,12 @@ Using forced rollback to switch back to a previous version might result in loss
 
 .Prerequisites
 
-* Before you can perform a rollback, you must have free disk space available on your persistent storage. {product-title} uses disk space to keep a copy of databases during the upgrade. If the disk space is not enough to store a copy and the upgrade fails, you will not be able to roll back to an earlier version.
+* **Enough disk space**: Before you can perform a rollback, you must have free disk space available on your persistent storage. {product-title} uses disk space to keep a copy of databases during the upgrade. If the disk space is not enough to store a copy and the upgrade fails, you will not be able to roll back to an earlier version.
+* **Internal database rollback (4.8 to earlier)**: If you are rolling back from {product-title-short} 4.8 to an earlier version and use the internal database (`central-db`), you must first restore the database from a PostgreSQL 13 backup.
+{product-title-short} creates this backup during the upgrade.
+
+** To restore the database, add the `RESTORE_BACKUP=true` and `FORCE_OLD_BINARIES=true` environment variables to the `central-db` and `init-db` containers of the `central-db` component.
+** For details on injecting environment variables, see "Injecting an environment variable into the Central deployment" in the additional resources section.
```

Comment:
You can drop the "additional resources section" based on this guidance:

![Screenshot From 2025-07-08 12-32-21](https://github.com/user-attachments/assets/3b8db707-99bb-4bdc-8ebe-993104069ab5)